recipes/implot/all/conanfile.py

```diff
@@ -59,4 +59,8 @@ def package(self):
         cmake.install()
 
     def package_info(self):
-        self.cpp_info.libs = ["implot"]
+        self.cpp_info.libs = tools.collect_libs(self)
```

Comment:
What triggered you to change this line?
```suggestion
        self.cpp_info.libs = ["implot"]
```

Reply:
Sorry its my mistake.
When I refered other recipe, I overwrited cpp_info.libs.

Reply:
No problem.
After fixing something, I am also guilty of forgetting to revert unneeded changes.